save_minifigure_prices.py

```diff
@@ -0,0 +1,30 @@
+import auth
+import database_handling as db
+from datetime import datetime
+
+def generate_minifigure_pricelist(minifigure_id):
+    client = auth.auth_on_bricklink()
+    url = 'https://api.bricklink.com/api/store/v1/items/minifig/'+minifigure_id+'/price?new_or_used=U&region=europe'
```

Comment:
maybe you can extract a base url like 
https://api.bricklink.com/api/store/v1 to a config file
would be easier to read